docker/auto-setup.sh

```diff
@@ -4,43 +4,91 @@ set -eu -o pipefail
 
 # === Auto setup defaults ===
 
-: "${DB:=cassandra}"
 : "${SKIP_SCHEMA_SETUP:=false}"
 : "${SKIP_DB_CREATE:=false}"
 
-# Cassandra
-: "${KEYSPACE:=temporal}"
+deprecate() {
+	if [ -n ${!1} ]; then
+		echo "The ${1} environment variable is deprecated. Please set ${2}=${!1} instead." >&2
+		export $2=${!1}
+	fi
+}
 
+# Cassandra
+deprecate KEYSPACE CASSANDRA_KEYSPACE
+: "${CASSANDRA_KEYSPACE:=temporal}"
 : "${CASSANDRA_SEEDS:=}"
 : "${CASSANDRA_PORT:=9042}"
 : "${CASSANDRA_USER:=}"
 : "${CASSANDRA_PASSWORD:=}"
-: "${CASSANDRA_TLS_ENABLED:=}"
-: "${CASSANDRA_CERT:=}"
-: "${CASSANDRA_CERT_KEY:=}"
-: "${CASSANDRA_CA:=}"
+deprecate CASSANDRA_TLS_ENABLED CASSANDRA_ENABLE_TLS
+: "${CASSANDRA_ENABLE_TLS:=}"
+deprecate CASSANDRA_CERT CASSANDRA_TLS_CERT
+: "${CASSANDRA_TLS_CERT:=}"
+deprecate CASSANDRA_CERT_KEY CASSANDRA_TLS_KEY
+: "${CASSANDRA_TLS_KEY:=}"
+deprecate CASSANDRA_CA CASSANDRA_TLS_CA
+: "${CASSANDRA_TLS_CA:=}"
 : "${CASSANDRA_REPLICATION_FACTOR:=1}"
 
-# MySQL/PostgreSQL
-: "${DBNAME:=temporal}"
-: "${VISIBILITY_DBNAME:=temporal_visibility}"
-: "${DB_PORT:=3306}"
+export CASSANDRA_SEEDS CASSANDRA_PORT CASSANDRA_USER CASSANDRA_PASSWORD CASSANDRA_KEYSPACE \
+    CASSANDRA_ENABLE_TLS CASSANDRA_TLS_CERT CASSANDRA_TLS_KEY CASSANDRA_TLS_CA CASSANDRA_REPLICATION_FACTOR
 
-: "${MYSQL_SEEDS:=}"
-: "${MYSQL_USER:=}"
-: "${MYSQL_PWD:=}"
+# SQL
+: "${DB:=}"
+if [ -n "${DB}" ]; then
+    if [ ${DB} == "cassandra" ]; then
+        echo "The DB environment variable is deprecated. Please unset DB, cassandra is the default." >&2
+    else
+        deprecate DB SQL_PLUGIN
+    fi
+fi
+: "${SQL_PLUGIN:=}"
+deprecate POSTGRES_SEEDS SQL_HOST
+deprecate MYSQL_SEEDS SQL_HOST
+: "${SQL_HOST:=}"
+if [ $SQL_PLUGIN == "postgres12" ] || [ $SQL_PLUGIN == "postgres12_pgx" ]; then
```

Comment:
Honestly not sure if we should be defaulting ports, I highly doubt anyone needs/wants that personally.